common/src/main/proto/TransportMessages.proto

```diff
@@ -220,6 +220,7 @@ message PbRevive {
 message PbChangeLocationResponse {
   int32 status = 1;
   PbPartitionLocation location = 2;
+  bool available = 3;
```

Comment:
I suppose the 0.2 client just simply ignores it and still works well, right?

Reply:
This PR is OK, but I do find compatibility exceptions when testing 0.2.1 client with main server. I will investigate and send new PRs to solve.